src/__tests__/index.to-string-class-name.js

```diff
@@ -0,0 +1,47 @@
+/* eslint func-style:0, react/prop-types:0 */
+import React from 'react'
+import {render} from 'enzyme'
+import * as jestGlamorReact from 'jest-glamor-react'
+import glamorous from '../'
+
+expect.extend(jestGlamorReact.matcher)
+expect.addSnapshotSerializer(jestGlamorReact.serializer)
+
+beforeEach(() => {
+  glamorous.config.idCounter = 1
```

Comment:
This is the best way I could think of to avoid the glamorous component IDs from growing from every other test.